src/themes/dark.scss

```diff
@@ -15,10 +22,6 @@
 
   .wxc-old-icon { display: none; }
   .wxc-icon { display: block; }
-}
 
-.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-theme-dark {
-  .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-member-roster .title {
-    display: block;
-  }
+  .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-member-roster .title { display: block; }
```

Comment:
let's keep the separate block and put all temporary rules in that block (variables in one block, temporary rules in another block)